recipe/meta.yaml

```diff
@@ -1,17 +1,19 @@
 {% set name = "hatch-requirements-txt" %}
-{% set version = "0.3.0" %}
+{% set version = "0.4.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hatch_requirements_txt-{{ version }}.tar.gz
-  sha256: bb87ecee32e4ac05d09854ac3c279dd526fbd655154acd9cd10c2a4768a83669
+  sha256: 2c686e5758fd05bb55fa7d0c198fdd481f8d3aaa3c693260f5c0d74ce3547d20
 
 build:
+  # Technically we should be [py<361] for 0.4.1 but conda doesn't
+  # recognise 361 and skips all pythons...
   skip: True  # [py<36]
```

Comment:
You can set `python >=3.6.1` in host/run to cover this case.

Reply:
Ah, TIL!

Reply:
Heh, I suppose the linter whinging is a small price to pay for correctness!  😄

Reply:
No you cannot set this in host! This overrides the python set by the cbc. If you check your logs you got only one variant built as a result. You can add it as run_constrained though.

Reply:
_Dangnabbit!_. He's right!

That's an unexpected interaction with `cbc.yaml`.

I guess it also affects any other multi-variant name too.